### PR TITLE
Cfy 3040 agents update

### PR DESCRIPTION
**Pull request by konradxyz**

### CODE REVIEW

File: cloudify_agent/operations.py [hunk 1]
```diff
@@ -191,3 +200,83 @@ def _save_daemon(daemon):
...
+    env_broker_url = os.environ.get('CELERY_BROKER_URL')
+    os.environ['CELERY_BROKER_URL'] = broker_url
+    try:
+        celery_client = celery.Celery(broker=broker_url, backend=broker_url)
+        # This one will have to be modified for 3.2
+        app_conf = {
+            'CELERY_TASK_RESULT_EXPIRES': defaults.CELERY_TASK_RESULT_EXPIRES
+        }
+        celery_client.conf.update(**app_conf)
+        script_format = 'http://{0}/api/{1}/node-instances/{2}/install_agent.py'
```
Comment (konradxyz) [inline cloudify_agent/operations.py]: Use static url + inputs

File: cloudify_agent/operations.py [hunk 2]
```diff
@@ -191,3 +200,83 @@ def _save_daemon(daemon):
...
+        script_url = script_format.format(
+            new_agent['manager_ip'],
+            DEFAULT_API_VERSION,
+            ctx.instance.id
+        )
+        result = celery_client.send_task(
```
Comment (konradxyz) [inline cloudify_agent/operations.py]: Investigate context

File: cloudify_agent/operations.py [hunk 3]
```diff
@@ -191,3 +200,83 @@ def _save_daemon(daemon):
...
+            'script_runner.tasks.run',
+            args=[script_url],
+            queue=old_agent['queue']
+        )
+        timeout = 2 * 60
```
Comment (konradxyz) [inline cloudify_agent/operations.py]: Configurable as workflow parameter?

Comment (konradxyz) [inline cloudify_agent/operations.py]: 5 mins

**Comment by barakm** [comment]
@konradxyz what is the status here?

**Comment by konradxyz** [comment]
Some more work is required. I am closing it for now.

**Pull request closed without merging**